platformio.ini

```diff
@@ -21,35 +21,44 @@
 env_default = emonesp
 data_dir = src/data
 
+[common]
+version = -DBUILD_TAG=2.0.0
+#MQTT: https://github.com/knolleary/pubsubclient
+lib_deps = PubSubClient@2.6
+
 [env:emonesp]
-platform = espressif
+platform = espressif8266
```

Comment:
The ESP32 has now been released (very cool chip BTW  so the espressif platform was changed to espressif8266 (you get a warning when building with espressif)